site/tutorials/weather.md

```diff
@@ -1,36 +1,204 @@
+  "mark": "tick",
+  "encoding": {
+    "x": {"field": "precipitation", "type": "quantitative"}
+  }
+}
+</div>
+
+It looks as though precipitation is skewed towards lower precipitations. To better see this, we can create a histogram of the precipitation data. For this, we have to add an encoding channel for `y` that uses a special field `*` that is aggregated with `count`. just showing how often a a certain value was measured is not very helpful because they are continuous. To discretize the temperature values, set `"bin": true`.
+
+<div class="vl-example">
+{
+  "data": {"url": "data/seattle-weather.csv", "formatType": "csv"},
+  "mark": "bar",
+  "encoding": {
+    "x": {
+      "field": "precipitation",
+      "type": "quantitative",
+      "bin": true
+      },
+    "y": {
+      "field": "*",
+      "type": "quantitative",
+      "aggregate": "count"
+    }
+  }
+}
+</div>
+
+Next, let's look at how precipitation in Seattle changes throughout the year. Vega-Lite natively supports dates and discretization of dates when we set the type to `temporal`. For example, in the following plot, we sum up the precipitation for each month. To discretize the data into months, we set `"timeUnit": "month"`. Vega-Lite automatically sets the right axis format. You can customize the format as described in the [documentation for axes]({{site.baseurl}}/docs/axis.html).
+
+<div class="vl-example">
+{
+  "data": {"url": "data/seattle-weather.csv", "formatType": "csv"},
+  "mark": "line",
+  "encoding": {
+    "x": {"field": "date", "type": "temporal", "timeUnit": "month"},
+    "y": {
+      "field": "precipitation",
+      "type": "quantitative",
+      "aggregate": "sum"
+    }
+  }
+}
+</div>
+
+This chart shows that in Seattle the precipitation in the winter months is much higher than during the summer. You can now create similar charts for the other variables on your own!
+
+When looking at precipitation and temperature, we might want to aggregate by year and month rather than just month. This allows us to see seasonal trends but for each year separately. You can find more about [time units in the documentation]({{site.baseurl}}/docs/timeUnit.html). Note that we set the aggregation to `max`so that we see the maximum temperature in each month.
+
+<div class="vl-example">
+{
+  "data": {"url": "data/seattle-weather.csv", "formatType": "csv"},
+  "mark": "line",
+  "encoding": {
+    "x": {
+      "field": "date",
+      "type": "temporal",
+      "timeUnit": "yearmonth"
+    },
+    "y": {
+      "field": "temp_max",
+      "type": "quantitative",
+      "aggregate": "max"
+    }
+  },
+  "config": {
+    "unit": { "width": 300 }
+  }
+}
+</div>
+
+In this chart it looks as though the maximum temperature is increasing from year to year. To confirm this, let's create a chart that shows the mean of the maximum daily temperatures.
+
+<div class="vl-example">
+{
+  "data": {"url": "data/seattle-weather.csv", "formatType": "csv"},
+  "mark": "point",
```

Comment:
Line has the problem that we actually don't have much data in between. And we don't have point on line.